app/views/components/InputTextCharacterCount.scala.html

```diff
@@ -76,7 +76,7 @@
                 inputmode = inputMode,
                 prefix = prefix.map(prefix => PrefixOrSuffix(content = Text(prefix))),
                 suffix = suffix.map(suffix => PrefixOrSuffix(content = Text(suffix))),
-                classes = if(inputClass.isDefined) inputClass.get else "hmrc-js-character-count"
+                classes = "hmrc-js-character-count" + {if (inputClass.isDefined) {" " + inputClass.get} else ""}
```

Comment:
@jassalrichy could you do this with string interpolation?
s"hmrc-js-character-count ${if(inputClass.getOrElse("")}"